src/test/shell/bazel/remote/remote_execution_http_test.sh

```diff
@@ -76,6 +76,10 @@ EOF
     || fail "Failed to build //a:test without remote cache"
   cp -f bazel-bin/a/test ${TEST_TMPDIR}/test_expected
 
+  bazel clean --expunge
+  bazel build //a:test --remote_http_cache= \
```

Comment:
please make this its own test case.

Reply:
I have pushed a new version. Replaced the test with a unit test instead.